tests/console/consoletest_setup.pm

```diff
@@ -66,28 +66,18 @@ sub run() {
     select_console 'root-console';
 
     type_string "chown $username /dev/$serialdev\n";
-    # Export the existing status of running tasks and system load for future reference (fail would export it again)
-    script_run "ps axf > /tmp/psaxf.log";
-    script_run "cat /proc/loadavg > /tmp/loadavg_consoletest_setup.txt";
```

Comment:
please keep them here, we want to see them anyway not only when the test failed.